src/app/App.js

```diff
@@ -35,53 +46,56 @@ function App() {
   };
 
   useEffect(() => {
-    // const getUsers = async () => {
-
-    //   const data = await getDocs(usersCollectionRef);
-    //   console.log(data)
-    //   setUsers(data.docs.map((doc) => ({ ...doc.data(), id: doc.id })));
-    // };
-
-    // getUsers();
     dispatch(fetchTodos("123"));
   }, []);
 
+  useEffect(() => {
+    dispatch(fetchUsers("jackson"));
+  }, []);
+
   return (
-    <div className="App">
-      <input
-        placeholder="Name..."
-        onChange={(event) => {
-          setNewName(event.target.value);
-        }}
-      />
-      <input
-        type="number"
-        placeholder="Age..."
-        onChange={(event) => {
-          setNewAge(event.target.value);
-        }}
-      />
-      <button onClick={createUser}>Create User</button>
+    <div className={styles.App}>
+      <div className={styles.createUser}>
+        <Input
+          placeholder="Name..."
+          onChange={(event) => {
+            users.name(event.target.value);
+          }}
+        />
+        <Input
+          type="number"
+          placeholder="Age..."
+          onChange={(event) => {
+            users.age(event.target.value);
+          }}
+        />
+        <Button type="primary" onClick={createUser}>
```

Comment:
The way createUser is setup right now this won't be doing anything. The argument passed in the onClick is the React Click Event like so `onClick={(event) => createUser()}`